derive/src/lib.rs

```diff
@@ -82,7 +83,7 @@ use syn;
 /// #[allow(non_snake_case)]
 /// mod Person_fields {
 ///     use super::*;
-///     // represents the `name` field of `Person`
+///     // defines the `name` field of `PersonFields`
```

Comment:
This edit isn't right, please revert it. The type `name` here theoretically represents the field `name` in `Person`, and that's what I was conveying in the docs. Sure, mechanically it is a field in `PersonFields`, but that doesn't tell us *why* it exists.